spec/EmitterSpec.php

```diff
@@ -82,6 +82,13 @@ public function it_should_allow_you_to_emit_plain_events()
         $this->emit('event')->shouldReturnAnInstanceOf('League\Event\Event');
     }
 
+    public function it_should_allow_you_to_emit_events_with_additional_arguments()
+    {
+        $callback = function ($event, $test, $example) {};
```

Comment:
@ameech you could simply create your own assertion here by throwing an exception when `$test !== 'test'`.